tests/integration/models/application.spec.coffee

```diff
@@ -581,3 +581,58 @@ describe 'Application Model', ->
 					.then =>
 						promise = balena.models.application.isTrackingLatestRelease(@application.id)
 						m.chai.expect(promise).to.eventually.be.true
+
+	describe 'given a multicontainer application with a single offline device', ->
+
+		givenMulticontainerApplication()
+
+		itShouldBeAnApplicationWIthDeviceServiceDetails = (application) ->
+			m.chai.expect(_.omit(application, 'owns__device', 'commit')).to.deep.equal(_.omit(@application, 'owns__device', 'commit'))
```

Comment:
It's weird that you had to also omit the `commit`.
What values did each of the objects contained?

Reply:
@application contained commit with null (because this variable is initialised when application is created), but then with `givenMulticontainerApplication` we do:

```coffeescript
# Register an old & new release of this application
				Promise.mapSeries [
					resource: 'release'
					body:
						belongs_to__application: @application.id
						is_created_by__user: userId
						commit: 'old-release-commit'
						status: 'success'
						source: 'cloud'
						composition: {}
						start_timestamp: 1234
				,
					resource: 'release'
					body:
						belongs_to__application: @application.id
						is_created_by__user: userId
						commit: 'new-release-commit'
						status: 'success'
						source: 'cloud'
						composition: {}
						start_timestamp: 54321
				]
```

But @application still contains old data, so `commit field is different`

Reply:
Oh that makes sense now @karaxuna :+1: 
If you also agree, it would be nice add a separate `expect` for the `commit` value

Reply:
Good idea